connector_jira/models/project_task/importer.py

```diff
@@ -108,6 +111,35 @@ def parent(self, record):
     def backend_id(self, record):
         return {'backend_id': self.backend_record.id}
 
+    @mapping
+    def status(self, record):
+        status = record['fields'].get('status', {})
+        status_name = status.get('name')
+        if not status_name:
+            return {'stage_id': False}
+        project_binder = self.binder_for('jira.project.project')
+        project_id = project_binder.unwrap_binding(
+            self.options.project_binding
+        )
+        stage_id = self.env['project.task.type'].search(
```

Comment:
you get a stage record here, not an id. Also, stage.id will give you already False if stage is not found: no need for the IF below ;)

Reply:
As you wish :)) Changed